tests/integration/test_gcloud/test_functionality/test_day_wday.py

```diff
@@ -97,53 +78,31 @@ def test_day_wday(tags_to_apply, get_configuration, configure_environment,
     These tests verify the module starts to pull according to the day of the week
     or month and time.
     """
-    check_apply_test(tags_to_apply, get_configuration['tags'])
+    def get_next_scan(next_scan_time: str):
+        next_scan_time = next_scan_time_log.split(" ")
+        date = next_scan_time[0].split("/")
+        hour = next_scan_time[1].split(":")
```

Comment:
```suggestion
        date = next_scan_time[0].split('/')
        hour = next_scan_time[1].split(':')
```

Reply:
Done in commit [b855b5c](https://github.com/wazuh/wazuh-qa/pull/1621/commits/b855b5c20954769f6d12e6fbffa8bdcb7b22bf90)